Makefile

```diff
@@ -9,16 +9,13 @@ SOURCEDIR     = docs/source
 BUILDDIR      = docs
 MODULEDIR     = st3
 
-# Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-.PHONY: help Makefile source
+.PHONY: Makefile source clean
 
 source:
-	SPHINX_APIDOC_OPTIONS=members,no-special-members sphinx-apidoc --separate --force -o "$(SOURCEDIR)" "$(MODULEDIR)"
+	SPHINX_APIDOC_OPTIONS=members,no-special-members sphinx-apidoc --separate --force --no-toc -o "$(SOURCEDIR)/modules" "$(MODULEDIR)"
+
+html:
+	sphinx-build -M html "$(SOURCEDIR)" "$(BUILDDIR)"
```

Comment:
When you remove usages of variables, you should also remove their definitions in the beginning of the file.